packages/go/dawgs/drivers/neo4j/cypher.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/specterops/bloodhound/log"
 )
 
-func updateKey(identityKind graph.Kind, identityProperties []string, updateKinds graph.Kinds) string {
+func newUpdateKey(identityKind graph.Kind, identityProperties []string, updateKinds graph.Kinds) string {
```

Comment:
I have a suspicion that this `new` naming convention is maybe an artifact of testing a new implementation while preserving an old implementation. If that's not the case, I'm curious the reasoning for the naming convention

Reply:
Nope, this was a naming collision that was making me upset. `updateKey` was used both for function names and instantiation assignment. So now the function is `newUpdateKey` to differentiate.